documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -1445,6 +1445,27 @@ or for the whole predicate with a call to `.boost(...)`
 after `.circle(...)`/`.boundingBox(...)`/`.polygon(...)`.
 
 
+[[search-dsl-predicate-not]]
+== `not`: negating another predicate
```

Comment:
Can you move this right after the documentation for the `and`/`or` predicates, please?